src/app/data/components/data-browser/data-browser.component.html

```diff
@@ -842,9 +786,9 @@
                         Clear filter
                     </button>
                 </div>
-                
 
-                <div data-tip="Change the data browser configurations" class="tooltip tooltip-left relative flex items-center"
+                <<<<<<< HEAD <div data-tip="Change the data browser configurations"
```

Comment:
i think i f... something up with the merge :D <<<<<<< HEAD shouldn't be there

Reply:
Removed it.